thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -695,7 +695,12 @@ def setup_compile(self, model):
                 executors.insert(0, transformer_engine_ex)
                 transforms.insert(0, TransformerEngineTransform())
 
-            if "dynamo" in self.compile:
+            if "jit" in self.compile:
+                jit_options = {}
+                jit_options["fp8_shard_intermediate_activation"] = self.fp8_shard_intermediate_activation
```

Comment:
This option was part of TE v1 executor and shouldn't have any effect now that we have removed it.